docs/snippets/react/button-story-click-handler-simple-docs.js.mdx

```diff
@@ -1,5 +1,17 @@
 ```js
 // Button.stories.js | Button.stories.jsx | Button.stories.ts | Button.stories.tsx
 
-export const Text = ({ label, onClick }) => <Button label={label} onClick={onClick} />;
-```
\ No newline at end of file
+import React from 'react';
+
+import { action } from '@storybook/addon-actions';
+
+import Button from './Button';
```

Comment:
```suggestion
import { Button } from './Button';
```

Reply:
I'm good with it. Thanks for taking a closer look at it.